app/views/play/level/HintsView.coffee

```diff
@@ -33,6 +35,16 @@ module.exports = class HintsView extends CocoView
     @listenTo(@hintsState, 'change', debouncedRender)
     @listenTo(@state, 'change:hintIndex', @updateHint)
     @listenTo(@hintsState, 'change:hidden', @visibilityChanged)
+    @showAiBotHelp = false
+    console.log("???Kjla,", @aceConfig.levelChat)
+    if @aceConfig.levelChat != 'none'
```

Comment:
i think we have this logic in some other file too, can we refactor it to lib/util to like `shouldShowAiBotHelp`

Reply:
ya sure